cpp/benchmarks/CMakeLists.txt

```diff
@@ -56,6 +55,14 @@ function(ConfigureBench CMAKE_BENCH_NAME)
         PRIVATE cudf_benchmark_common cudf_datagen benchmark::benchmark_main)
 endfunction()
 
+function(ConfigureNVBench CMAKE_BENCH_NAME)
+    add_executable(${CMAKE_BENCH_NAME} ${ARGN})
+    set_target_properties(${CMAKE_BENCH_NAME}
```

Comment:
I don't know if we want to place benchmarks in different directories based on if they used `gbench` or `nvbench`. To the person running the benchmark it is an implementation detail, and they would expect all benchmarks to be located in the same folder.

For that reason I think placing all the benchmarks in either the `gbenchmark` or `benchmark` folder seems more reasonable to me

Reply:
Right, I hesitate on this different directories idea also. The `gbenchmark` naming basically implies that google benchmark is applied. So my only concern is that creating a `nvbench` code in `gbenchmark` would be misleading.

Reply:
> So my only concern is that creating a nvbench code in gbenchmark would be misleading.

I think most developers won't remember what benchmarks use nvbench, and will be confused on the split.

This might be the perfect time to have all benchmarks in `<build_dir>/benchmarks` :)